plugins/org.eclipse.xtend.ide.common/src/org/eclipse/xtend/ide/common/highlighting/XtendHighlightingCalculator.java

```diff
@@ -384,6 +384,7 @@ public void acceptSemanticLineBreak(int charCount, RichStringLiteral origin, boo
 		@Override
 		public void acceptTemplateLineBreak(int charCount, RichStringLiteral origin) {
 			resetCurrentOffset(origin);
+			acceptor.addPosition(currentOffset, charCount, TEMPLATE_LINE_BREAK);
```

Comment:
This breaks the Xtend highlighting tests and I think the linebreak will be highlighted twice in some cases, now (see acceptSemanticLineBreak)